<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ReferenceButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference Structure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ReferenceStructurePath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>461</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;reference&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectPpbs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Pdbs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectPqrs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Pqrs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputPath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>461</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisplayX">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>DisplayX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DisplayXValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1175</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DisplayYValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisplayY">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>DisplayY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apbsInButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>apbs.in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apbsInText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>391</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>apbs.in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunImageCreator">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Run ImageCreator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ColumnsValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Columns">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Columns</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ImageCreatorQuality_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImageCreatorQuality">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Console">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pdb2pqr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>pdb2pqr</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="vmd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>vmd</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="doImageCreator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>imageCreator</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="doMovieCreator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>movieCreator</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="doCreateTrainingsSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>trainingsSet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutputPath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>461</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xRot">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xRotButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>xRotation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yRot">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xRotButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>yRotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zRotButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>zRotation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zRot">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xTrans">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zTrans">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xTransButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>xTranslation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yTransButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>yTranslation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yTrans">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zTransButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>zTranslation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewPointsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ViewPoints</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ViewPoints">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>461</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ViewPoints</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FontSizeValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FontSizeButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>FontSize</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
